cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -122,6 +122,9 @@ Instead, the CDI uses the default storage class to allocate scratch space.
 $ oc patch storageclass <STORAGE_CLASS_NAME> -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
 ----
 
+* The {CNVProductName} {CNVVersion} web console includes *slirp* as an option when you add a NIC to a virtual machine, but *slirp* is not a valid NIC type. Do not select *slirp* when adding a NIC to a virtual machine.
```

Comment:
I actually think this should be 
OpenShift Container Platform 4.4 web console 
and not CNV 
just like we have it in the installation guide: "You can use the OpenShift Container Platform 4.3 web console to subscribe...."

Reply:
Thanks Nelly. Updated